examples/bzlmod_build_file_generation/BUILD.bazel

```diff
@@ -45,12 +45,10 @@ modules_mapping(
 # When you are using gazelle you need to run this target first.
 gazelle_python_manifest(
     name = "gazelle_python_manifest",
+    manifest = ":gazelle_python.yaml",
+    manifest_windows = ":gazelle_python_windows.yaml",
```

Comment:
IMHO we should use the filtering functionality in the `modules_mapping` so that we can exclude and make the manifest the same on all platforms.

We don't support loading `gazelle_python_windows.yaml` anyway. Right now the differences are minor and I think if the user intends to use `colorama` they should add it to the `requirements.in` file and then it will be present in the manifest on all platforms.

Diff for reference:
```
>     colorama: colorama
>     colorama.ansi: colorama
>     colorama.ansitowin32: colorama
>     colorama.initialise: colorama
>     colorama.win32: colorama
>     colorama.winterm: colorama
235d240
<     lazy_object_proxy.cext: lazy_object_proxy
```

Reply:
Ah, I didn't know about the modules_mapping filtering. Let me take a look at that.

Reply:
Thank you! That's a way better solution than a Windows manifest.